README.md

```diff
@@ -1,17 +1,154 @@
 # Simmy
-Simmy is a chaos-engineering and fault-injection tool, integrating with the Polly resilience project for .NET.  It is expected to release in early 2019 and will work with Polly v7.0.
+Simmy is a [chaos-engineering](http://principlesofchaos.org/) and fault-injection tool, integrating with the Polly resilience project for .NET.  It is expected to release in early 2019 and will work with Polly v7.0.
 
-## Concept
+# Motivation
 
-_These grabs of slides from our recent conference presentations represent concept - final syntax will be confirmed in documentation but all features are in scope for launch_
+There are a lot of questions when it comes to chaos-engineering and make sure that actually my system is ready to face the worst possibles scenarios.
 
-1 ![simmyslidesnip1](https://user-images.githubusercontent.com/9608723/52367255-8ab7fd80-2a43-11e9-9d55-d9e8abfe14a5.GIF)
+* My system is really resilient enough?
+* Am I really handling the right exceptions/scenarios?
+* Why should I wait for a handled or even and unhandled exception happens in my production environment?
```

Comment:
misspelled over "and unhandled..." should be "an unhandled..."

Reply:
Self review thing is awesome

Reply:
oops first time I do it, just making sure I don't miss this guy